src/Secret.php

```diff
@@ -83,14 +102,22 @@ public function getSecret($secret, string $secretVersion = null) {
             ),
             $response->contentType ?? null,
         );
+
+        // Set the secret to expire in one day and store it in cache
+        $secretCache->expiresAt(new \DateTime('next month'));
+        $secretCache->set($secretEntity);
+        $cache->save($secretCache);
```

Comment:
do you know if this is encrypted at rest?

Reply:
It is not, but neither was the original implementation.

Reply:
It kind of invalidates the whole point of keyvault to not encrypt it... perhaps we could explore another caching solution that isn't on the filesystem?

Reply:
I misunderstood - this isn't a file cache it's an in-memory cache. I'm cool with that.